docs/build-instructions/freebsd.md

```diff
@@ -12,13 +12,31 @@ FreeBSD -RELEASE 64-bit is the recommended platform.
 
 ## Instructions
 
+If you have problems with permissions don't forget to prefix with `sudo`
```

Comment:
`sudo script/build` usually results in a lot of problems when running Atom as a normal user - are you sure about this?